02-IDE_exercises/solutions/08b-project/words_in_both.py

```diff
@@ -2,12 +2,18 @@
 # Date: 11/14/2020
 # Description: Takes in two strings and returns a set of common words
 
-def words_in_both(string_1, string_2):
+def words_in_both(string_1: str, string_2: str) -> set[str]:
```

Comment:
incorrect type hint for `set`. Should be `from typing import Set` and the n `Set[str]`